client/src/pages/Home/HeroBanner/HeroBanner.tsx

```diff
@@ -0,0 +1,110 @@
+import { FaCheckCircle, FaChevronRight } from 'react-icons/fa';
+import { Button } from '@/components/ui/button.tsx';
+import { TypewriterEffect } from '@/components/ui/typewriter-effect.tsx';
+import heroImg from '../../../assets/HeroImage.jpg';
+import ChatBox from './ChatBox.tsx';
+
+export default function HeroBanner() {
+  const words = [
+    {
+      text: 'Exam',
+      className: 'text-blue-500 text-[80px] dark:text-blue-500',
+    },
+    {
+      text: 'Archive',
+      className: 'text-blue-500 text-[80px] dark:text-blue-500',
+    },
+    {
+      text: 'is',
+    },
+    {
+      text: 'your',
+    },
+    {
+      text: 'one',
+    },
+    {
+      text: 'stop',
+    },
+    {
+      text: 'solution',
+    },
+    {
+      text: 'for',
+    },
+    {
+      text: 'PYQs',
+      className: 'text-[45px] font-bold',
+    },
+  ];
+
+  return (
+    <section className=" bg-[#b1d8fc] rounded-b-[80px]">
+      <div className="w-[1280px] min-h-[670px] h-full mx-auto flex flex-row">
+        <div className="w-[50%] flex flex-col justify-center px-4">
+          <div className="flex flex-col gap-y-6">
+            <TypewriterEffect words={words} />
+            <div>
+              <p className="flex flex-row items-center gap-x-2 text-gray-500">
+                <FaCheckCircle className="text-lg" />
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Soluta,
+                saepe?
+              </p>
+              <p className="flex flex-row items-center gap-x-2 text-gray-500">
+                <FaCheckCircle className="text-lg" />
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Soluta,
+                saepe?
+              </p>
+            </div>
+            <Button className="max-w-[300px] rounded-3xl">
+              Get Started
+              <FaChevronRight className="w-6" />
+            </Button>
+          </div>
+        </div>
+        <div className="w-[50%] flex flex-col relative justify-center">
+          <img
+            src={heroImg}
+            alt="Hero Image"
+            className="rounded-tl-[80px] rounded-br-[80px]"
+          />
+          <ChatBox
```

Comment:
Avoid rendering same components multiple times. Grab all props in an array (array of props) and just map over that array.